db/migrate/20210813115346_add_expire_to_absence.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddExpireToAbsence < ActiveRecord::Migration[6.0]
+  def change
+    add_column :absences, :expired, :boolean
```

Comment:
C’est un peu un réflexe pavlovien, mais j’aime bien les colonnes nonnull :)
```suggestion
    add_column :absences, :expired, :boolean, null: false, default: false
```